### PR TITLE
Add a rough script to plot frame times

### PR DESCRIPTION
**Pull request by brianwatling**

### CODE REVIEW

File: scripts/plot_frame_times.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```
Comment (kuhar) [inline scripts/plot_frame_times.sh]: Use `#!/usr/bin/env bash`

File: scripts/plot_frame_times.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+#
+# This script can be used to plot frame time layer output with gnuplot. A PNG is
+# generated containing a CDF of frame times in milliseconds.
+#
+# Usage:
+#   plot_fram_times.sh frame_time_a.txt frame_time_b.txt
```
Comment (kuhar) [inline scripts/plot_frame_times.sh]: Can you mention that what the default output filename is?

File: scripts/plot_frame_times.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
```
Comment (kuhar) [inline scripts/plot_frame_times.sh:9]: Consider adding `set -e`

**Review by kuhar — APPROVED** [review]
Very useful. LGTM modulo some nits.

File: scripts/plot_frame_times.sh (hunk 1 above)
Comment (brianwatling) [reply scripts/plot_frame_times.sh]: Done

File: scripts/plot_frame_times.sh (hunk 2 above)
Comment (brianwatling) [reply scripts/plot_frame_times.sh]: Done

File: scripts/plot_frame_times.sh (hunk 3 above)
Comment (brianwatling) [reply scripts/plot_frame_times.sh:9]: Done

**Pull request merged**